### PR TITLE
Flux 'getRecord' and 'getColumn' functions

### LINKED ISSUE #874: Document 'experimental.alignTime' function
PR: https://github.com/influxdata/flux/pull/2671

### LINKED ISSUE #932: Document 'findColumn` and `findRecord' functions
PR: https://github.com/influxdata/flux/pull/2695

##### Relevant URLs
- https://v2.docs.influxdata.com/v2.0/reference/flux/stdlib/built-in/transformations/stream-table/
- https://v2.docs.influxdata.com/v2.0/query-data/flux/scalar-values/

### PR DESCRIPTION
**Pull request by sanderson**
Closes #932 

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/v2.0/reference/flux/stdlib/built-in/transformations/stream-table/findcolumn.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  fn: (key) => key._field == "fieldName")
+  column: "_value"
+)
+```
+
+## Parameters
+
+### fn
+A predicate function for matching keys in a table's group key.
+It expects a `key` argument which represents a group key in the input stream.
```
Comment (kelseiv) [inline content/v2.0/reference/flux/stdlib/built-in/transformations/stream-table/findcolumn.md:33]: Expects

Comment (kelseiv) [reply content/v2.0/reference/flux/stdlib/built-in/transformations/stream-table/findcolumn.md:33]: that represents

File: content/v2.0/reference/flux/stdlib/built-in/transformations/stream-table/findrecord.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  fn: (key) => key._field == "fieldName",
+  idx: 0
+)
+```
+
+## Parameters
+
+### fn
+A predicate function for matching keys in a table's group key.
+It expects a `key` argument which represents a group key in the input stream.
```
Comment (kelseiv) [inline content/v2.0/reference/flux/stdlib/built-in/transformations/stream-table/findrecord.md:32]: Expects

Comment (kelseiv) [reply content/v2.0/reference/flux/stdlib/built-in/transformations/stream-table/findrecord.md:32]: that repr...

File: content/v2.0/query-data/flux/scalar-values.md [hunk 3]
```diff
@@ -38,61 +38,32 @@ _The samples on this page use the [sample data provided below](#sample-data)._
...
-of tables piped-forward into `tableFind()`.
+Flux functions do not guarantee table order.
+`findColumn()` and `findRecord` extract only the **first** table that matches the `fn` predicate.
+To extract the correct table, be very specific in your predicate function or
+filter and transform your data to minimize the number of tables piped-forward into the functions.
 {{% /note %}}
 
-## Extract a column from the table
+## Extract a column
 Use the [`getColumn()` function](/v2.0/reference/flux/stdlib/built-in/transformations/stream-table/getcolumn/)
```
Comment (adrian-thurston) [inline content/v2.0/query-data/flux/scalar-values.md:56]: `findColumn`?

File: content/v2.0/query-data/flux/scalar-values.md [hunk 4]
```diff
@@ -121,19 +93,18 @@ SFOTemps[2]
 // Returns 66.3
 ```
 
-## Extract a row from the table
+## Extract a row
 Use the [`getRecord()` function](/v2.0/reference/flux/stdlib/built-in/transformations/stream-table/getrecord/)
```
Comment (adrian-thurston) [inline content/v2.0/query-data/flux/scalar-values.md:97]: `findRecord`?

**Comment by sanderson** [comment]
@adrian-thurston Will these be merged into Flux master before the next release?

**Comment by adrian-thurston** [comment]
> @adrian-thurston Will these be merged into Flux master before the next release?

@sanderson Sorry, I'm not sure why I didn't see this tag.

They are in the also-w branch. That should be merged very soon, assuming no further issues, but I'm not sure exactly when.

**Pull request merged**